lib/generators/templates/fuel.rb

```diff
@@ -54,4 +54,9 @@
   # Add Twitter
   #config.twitter = true
 
+  # AWS S3 SETTINGS
+  config.aws_bucket = ENV['AWS_BUCKET']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.